### PR TITLE
SBERDOMA-128 fix password length on register/password reset form

### PR DESCRIPTION
**Pull request by Dimitreee**

### CODE REVIEW

**Review by dkoviazin — APPROVED** [review]
OK

File: apps/condo/pages/auth/change-password.tsx [hunk 1]
```diff
@@ -82,7 +80,7 @@ const ChangePasswordPage: AuthPage = () => {
                             message: PleaseInputYourPasswordMsg,
                         },
                         {
-                            min: 7,
+                            min: 8,
```
Comment (AntonAL) [inline apps/condo/pages/auth/change-password.tsx:83]: Extract it to global constant

File: apps/condo/pages/auth/register.tsx [hunk 2]
```diff
@@ -467,7 +464,7 @@ const RegisterForm = ({ onFinish }): React.ReactElement<IRegisterFormProps> => {
                             message: PleaseInputYourPasswordMsg,
                         },
                         {
-                            min: 7,
+                            min: 8,
```
Comment (AntonAL) [inline apps/condo/pages/auth/register.tsx:467]: Extract it to global constant

**Review by AntonAL — COMMENTED** [review]
Password length is also checked in `apps/condo/domains/user/schema/RegisterNewUserService.js`

```
if (userData.password.length < 7) {
    throw new Error(`${MIN_PASSWORD_LENGTH_ERROR}] Password length less then 7 character`)
}
```

File: apps/condo/domains/user/schema/RegisterNewUserService.js [hunk 3]
```diff
@@ -83,8 +84,8 @@ const RegisterNewUserService = new GQLCustomSchema('RegisterNewUserService', {
                     throw new Error(`${EMAIL_ALREADY_REGISTERED_ERROR}] User with this email is already registered`)
                 }
 
-                if (userData.password.length < 7) {
-                    throw new Error(`${MIN_PASSWORD_LENGTH_ERROR}] Password length less then 7 character`)
+                if (userData.password.length < MIN_PASSWORD_LENGTH) {
+                    throw new Error(`${MIN_PASSWORD_LENGTH_ERROR}] Password length less then 8 character`)
```
Comment (dkoviazin) [inline apps/condo/domains/user/schema/RegisterNewUserService.js:88]: 8

**Review by dkoviazin — APPROVED** [review]
Ok

**Review by toplenboren — APPROVED** [review]

**Comment by Dimitreee** [comment]
> Password length is also checked in `apps/condo/domains/user/schema/RegisterNewUserService.js`
> 
> ```
> if (userData.password.length < 7) {
>     throw new Error(`${MIN_PASSWORD_LENGTH_ERROR}] Password length less then 7 character`)
> }
> ```

fixed

**Pull request merged**